### ISSUE TITLE
Object detector vs pedestrian detector?? gcc version??

### ISSUE THREAD

**Issue by msha096** [hasanirtiza/Pedestron#64]
Thanks for your great work!

I have a question, what parameters did you change when applying the object detector (RetinaNet) to detect pedestrian on the CityPersosnd dataset in addition to the image scale? The anchor ratio? Since the bounding box ratio is fixed at 0.41. Also, what is the output size of RetinaNet...

Thanks!!

Also, I tried to run the setup.py, however it seems my gcc version(7.5) does not work. 

```
(base) ➜  Pedestron-master python tools/train.py configs/elephant/citypersons/retinanet_ResNext101.py
No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda-10.0'
Traceback (most recent call last):
  File "tools/train.py", line 7, in <module>
    from mmdet import __version__
ModuleNotFoundError: No module named 'mmdet'
```

I could not install gcc 7.3 or other versions. I got this:

Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package gcc-7.3
E: Couldn't find any package by glob 'gcc-7.3'
E: Couldn't find any package by regex 'gcc-7.3'

Anyone can help??

**Comment by hasanirtiza** [comment]
If you want to see what parameters did we change for RetinaNet, you can have a look at the configs file. For pedestrian detection on Cityperson using RetinaNet-ResNeXt101, configs file are in `configs/elephant/cityperson/retinanet_ResNeXt101.py ` you can compare it with generic object detection configs in `configs/retinanet_x101_64x4d_fpn_1x.py`. 

No, we did not opt for fixed ratio and I believe besides some minor modifications we did not change too much in the configs. 

Regarding the error trail ? Did you manage to compile Pedestron and did you have a look at the [installation file](https://github.com/hasanirtiza/Pedestron/blob/master/INSTALL.md) or [colab demo](https://github.com/hasanirtiza/Pedestron/blob/master/colab/PedestronColab.ipynb).  
May be try lowering your gcc version ?

**Comment by msha096** [comment]
> If you want to see what parameters did we change for RetinaNet, you can have a look at the configs file. For pedestrian detection on Cityperson using RetinaNet-ResNeXt101, configs file are in `configs/elephant/cityperson/retinanet_ResNeXt101.py ` you can compare it with generic object detection configs in `configs/retinanet_x101_64x4d_fpn_1x.py`.
> 
> No, we did not opt for fixed ratio and I believe besides some minor modifications we did not change too much in the configs.
> 
> Regarding the error trail ? Did you manage to compile Pedestron and did you have a look at the [installation file](https://github.com/hasanirtiza/Pedestron/blob/master/INSTALL.md) or [colab demo](https://github.com/hasanirtiza/Pedestron/blob/master/colab/PedestronColab.ipynb).
> May be try lowering your gcc version ?

Thanks for your reply. Before I asking the question I have already read the instruction and installed the prerequisites. 
Now I have downgraded gcc version to 4.9 successfully, but I still have the same problem when I run` python setup.py develop`

```
/home/mingzhi/anaconda3/envs/pedestron/lib/python3.7/site-packages/torch/include/c10/util/Optional.h: In member function ‘constexpr T&& c10::optional<T>::operator*() const && [with T = int]’:
/home/mingzhi/anaconda3/envs/pedestron/lib/python3.7/site-packages/torch/include/c10/util/Optional.h:473:3: warning: control reaches end of non-void function [-Wreturn-type]
   }
   ^
error: command 'gcc' failed with exit status 1
```

**Comment by msha096** [comment]
> > If you want to see what parameters did we change for RetinaNet, you can have a look at the configs file. For pedestrian detection on Cityperson using RetinaNet-ResNeXt101, configs file are in `configs/elephant/cityperson/retinanet_ResNeXt101.py ` you can compare it with generic object detection configs in `configs/retinanet_x101_64x4d_fpn_1x.py`.
> > No, we did not opt for fixed ratio and I believe besides some minor modifications we did not change too much in the configs.
> > Regarding the error trail ? Did you manage to compile Pedestron and did you have a look at the [installation file](https://github.com/hasanirtiza/Pedestron/blob/master/INSTALL.md) or [colab demo](https://github.com/hasanirtiza/Pedestron/blob/master/colab/PedestronColab.ipynb).
> > May be try lowering your gcc version ?
> 
> Thanks for your reply. Before I asking the question I have already read the instruction and installed the prerequisites.
> Now I have downgraded gcc version to 4.9 successfully, but I still have the same problem when I run` python setup.py develop`
> 
> ```
> /home/mingzhi/anaconda3/envs/pedestron/lib/python3.7/site-packages/torch/include/c10/util/Optional.h: In member function ‘constexpr T&& c10::optional<T>::operator*() const && [with T = int]’:
> /home/mingzhi/anaconda3/envs/pedestron/lib/python3.7/site-packages/torch/include/c10/util/Optional.h:473:3: warning: control reaches end of non-void function [-Wreturn-type]
>    }
>    ^
> error: command 'gcc' failed with exit status 1
> ```


> > If you want to see what parameters did we change for RetinaNet, you can have a look at the configs file. For pedestrian detection on Cityperson using RetinaNet-ResNeXt101, configs file are in `configs/elephant/cityperson/retinanet_ResNeXt101.py ` you can compare it with generic object detection configs in `configs/retinanet_x101_64x4d_fpn_1x.py`.
> > No, we did not opt for fixed ratio and I believe besides some minor modifications we did not change too much in the configs.
> > Regarding the error trail ? Did you manage to compile Pedestron and did you have a look at the [installation file](https://github.com/hasanirtiza/Pedestron/blob/master/INSTALL.md) or [colab demo](https://github.com/hasanirtiza/Pedestron/blob/master/colab/PedestronColab.ipynb).
> > May be try lowering your gcc version ?
> 
> Thanks for your reply. Before I asking the question I have already read the instruction and installed the prerequisites.
> Now I have downgraded gcc version to 4.9 successfully, but I still have the same problem when I run` python setup.py develop`
> 
> ```
> /home/mingzhi/anaconda3/envs/pedestron/lib/python3.7/site-packages/torch/include/c10/util/Optional.h: In member function ‘constexpr T&& c10::optional<T>::operator*() const && [with T = int]’:
> /home/mingzhi/anaconda3/envs/pedestron/lib/python3.7/site-packages/torch/include/c10/util/Optional.h:473:3: warning: control reaches end of non-void function [-Wreturn-type]
>    }
>    ^
> error: command 'gcc' failed with exit status 1
> ```

The above problem is solved when I change PyTorch from 1.6 to 1.2... Aha
